src/cli/argv_parser.ts

```diff
@@ -227,7 +233,21 @@ const ArgvParser = {
       /* eslint-enable no-console */
     })
 
-    program.parse(argv)
+    try {
+      program.parse(argv)
+      // @ts-expect-error
```

Comment:
What does this comment do?

Reply:
The next line gives a compile error. `catch (commanderError: CommanderError)` is not allowed, you're only alowed to catch `any` or `unknown`. 

Using `@ts-expect-error` will ignore the error (like `@ts-ignore-error`), but is a bit better in that it _will give a error when the next line does not give a compile error_. For example, if you forget to remove `@ts-ignore-error` after a refactor.

I'd say: use `@ts-expect-error` with caution and never use `@ts-ignore-error`.